client/app/pages/queries/QueryView.jsx

```diff
@@ -209,10 +207,10 @@ function QueryView(props) {
               <QueryViewButton
                 className="icon-button m-r-5 hidden-xs"
                 title="Toggle Fullscreen"
-                type={fullscreen ? "primary" : "default"}
+                type="default"
                 shortcut="alt+f"
                 onClick={toggleFullscreen}>
-                <i className="zmdi zmdi-fullscreen" />
+                <Icon type={fullscreen ? "fullscreen-exit" : "fullscreen"} />
```

Comment:
Switched to Ant's icons. If we like it, we should use it in other places where we have "fullscreen" toggle.

Reply:
I'm leaned towards the zmdi one, but I think  I'm biased anyway.